com.tradingview.tradingview.yaml

```diff
@@ -0,0 +1,53 @@
+id: com.tradingview.tradingview
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+separate-locales: false
+command: tradingview
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
```

Comment:
why does this need sound and device=all?

Reply:
Tradingview has functions that require a camera and microphone, and also need to play the video from the chat room.